### PR TITLE
Updated budspencer theme

### PR DESCRIPTION
**Pull request by tannhuber**
I've made some small changes in order to suppress non-relevant output. The README was updated as well.

### CODE REVIEW

File: themes/budspencer/README.md [hunk 1]
```diff
@@ -195,6 +216,17 @@ Just type `wt <title>`.
...
+## Feedback
+
+Give me feedback (<sepp.tannhuber@yahoo.de>) if you ...
+* ... like budspencer.
+* ... have ideas how I could improve budspencer.
+* ... have problems with budspencer. 
+* ... have questions concerning budspencer.
+
+Don't give me feedback if you ...
+* ... don't like budspencer. Use a different theme instead.
```
Comment (bpinto) [inline themes/budspencer/README.md]: lmao

File: themes/budspencer/README.md [hunk 2]
```diff
@@ -195,6 +216,17 @@ Just type `wt <title>`.
 
 - vi REPLACE mode, as soon as REPLACE mode is implemented within fish
 
+## Feedback
+
+Give me feedback (<sepp.tannhuber@yahoo.de>) if you ...
```
Comment (bpinto) [inline themes/budspencer/README.md]: You sure you want to add your email like this? I'd rather be pinged on github.

File: themes/budspencer/fish_prompt.fish [hunk 3]
```diff
@@ -401,70 +424,90 @@ function s -d 'Create, delete or attach session'
...
+        for i in (seq (expr $num_items + 1))
+          tput cuu1
+          end
+          tput ed
+          tput cuu1
+      case 'e'
+        read -p 'echo -n (set_color -b $budspencer_colors[2] $budspencer_colors[8])" ✻ Erase [0"$last_item"] "(set_color -b normal $budspencer_colors[2])" "(set_color $budspencer_colors[8])' -n $input_length -l session_num
+        if [ (expr $num_items - $session_num) -gt 0 ]
+          __budspencer_erase_session -e $budspencer_sessions[(expr $num_items - $session_num)]
+          end
```
Comment (bpinto) [inline themes/budspencer/fish_prompt.fish]: Indentation issues.

File: themes/budspencer/fish_prompt.fish [hunk 4]
```diff
@@ -401,70 +424,90 @@ function s -d 'Create, delete or attach session'
...
+          for i in (seq (expr $num_items + 3))
+            tput cuu1
+              end
```
Comment (bpinto) [inline themes/budspencer/fish_prompt.fish]: Indentation issues.

File: themes/budspencer/fish_prompt.fish [hunk 5]
```diff
@@ -401,70 +424,90 @@ function s -d 'Create, delete or attach session'
...
+      echo -en $budspencer_cursors[3]
+      set input_length (expr length (expr $num_items - 1))
+      read -p 'echo -n (set_color -b $budspencer_colors[2] $budspencer_colors[8])" ✻ Attach [e|0"$last_item"] "(set_color -b normal $budspencer_colors[2])" "(set_color $budspencer_colors[8])' -n $input_length -l session_num
+      set pcount (expr $pcount - 1)
+      switch $session_num
+      case (seq 0 (expr $num_items - 1))
+        set argv[1] $budspencer_sessions[(expr $num_items - $session_num)]
+        for i in (seq (expr $num_items + 1))
+          tput cuu1
+          end
```
Comment (bpinto) [inline themes/budspencer/fish_prompt.fish]: Indentation issues.

File: themes/budspencer/fish_prompt.fish [hunk 6]
```diff
@@ -279,33 +297,36 @@ function m -d 'List bookmarks, jump to directory in list with m <number>'
...
-end
-for i in (seq $num_items)
-  if [ $PWD = $bookmarks[$i] ]
-    set_color $budspencer_colors[10]
-  else
-    if [ (expr \( $num_items - $i \) \% 2) -eq 0 ]
-      set_color normal
+  end
+  for i in (seq $num_items)
+    if [ $PWD = $bookmarks[$i] ]
```
Comment (bpinto) [inline themes/budspencer/fish_prompt.fish]: There are a few if/elses with indentation issues, I won't comment on all of them, but it's a good idea to check all of them.

Comment (tannhuber) [inline themes/budspencer/fish_prompt.fish]: Vim's autoindent still sucks with fish syntax. Thanks for checking this.

**Comment by bpinto** [comment]
Merged, thanks!

**Pull request closed without merging**